draft-patton-cfrg-vdaf.md

```diff
@@ -839,20 +839,24 @@ def vec_add(left, right):
 
 ## Pseudorandom Generators {#prg}
 
+> CP TODO Update the reference implementation to match.
+
 A pseudorandom generator (PRG) is used to expand a short, (pseudo)random seed
 into a long string of pseudorandom bits. A PRG suitable for this document
 implements the interface specified in this section. Concrete constructions are
 described in {{prg-constructions}}.
 
-PRGs are defined by a class `Prg` with a single parameter:
+PRGs are defined by a class `Prg` with the follwing associated parameter:
 
 * `SEED_SIZE: Unsigned` is the size of the input seed.
 
 A concrete `Prg` implements the following class method:
 
-* `Prg.expand(seed: Bytes, info: Bytes, length: Unsigned) -> Bytes` derives a
-  byte string from the given seed and info string. The length of the output MUST
-  be `length`.
+* `Prg(seed: Bytes, info: Bytes, length: Unsigned)` constructs an instance of
+  `Prg` from the given seed and info string.
+
+* `prg.next(length: Unsigned)` returns the next `length` bytes of output of the
+  PRG.
```

Comment:
Please add some info about what we expect from the output, e.g.
`
The bytes returned are uniformly pseudorandom when the PRG was constructed with a uniformly random seed.
`

Reply:
Done. I also stipulated that the random seed might be an output of a previous call to the PRG.